app/main/plugins/core/basic-apps/index.js

```diff
@@ -27,7 +28,10 @@ const fn = ({ term, actions, display }) => {
           event.preventDefault()
         }
       },
-      onSelect: () => openApp(app),
+      onSelect: () => {
+        app.selectCount = app.selectCount ? app.selectCount + 1 : 1;
```

Comment:
So, you can use the same logic, but move sorting to this file [before](https://github.com/TheConnMan/cerebro/blob/2d590903c15ceaf860945a0e760f773068623937/app/main/plugins/core/basic-apps/index.js#L14) `map`